doc/library-development/ADRs/reduce-packages.md

```
@@ -0,0 +1,112 @@
+| ndk  | -- | ndk |
+| ndk_cache_manager_test_suite  | -- | ndk_cache_manager_test_suite |
+| ndk_flutter  | -- | ndk_flutter |
+| nip07_event_signer  | -- | nip07_event_signer |
+| ndk_objectbox  | -- | ndk_objectbox |
+| ndk_rust_verifier  |  embedded => |  ndk (native assets) |
+| ndk_demo  | -- | ndk_demo |
+| sembast_cache_manager  | embedded => | ndk |
+| ndk_bip32_keys  |  embedded => | ndk | 
+
+changes:
+| previous package | operation | new structure |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)